test/docker_stdout_example_test.go

```diff
@@ -0,0 +1,33 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/docker"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDockerComposeStdoutExample(t *testing.T) {
+	dockerComposeFile := "../examples/docker-compose-stdout-example/docker-compose.yml"
+
+	// Build the Docker image.
```

Comment:
NIT: might want to mention that we do this as a separate step as it writes lots of stuff to `stdout` that we don't want to pollute the text we try to read in the next step.